packages/domain-picker/src/domain-picker/index.tsx

```diff
@@ -267,18 +274,25 @@ const DomainPicker: FunctionComponent< Props > = ( {
 			{ header && header }
 			{ showSearchField && (
 				<div className="domain-picker__search">
-					<div className="domain-picker__search-icon">
-						<Icon icon={ search } />
-					</div>
-					<TextControl
-						hideLabelFromVision
-						label={ label }
-						placeholder={ label }
-						onChange={ handleInputChange }
-						onBlur={ onDomainSearchBlurValue }
-						value={ domainSearch }
-						dir="ltr"
-					/>
+					{ /* Form is to allow hiding keyboard on "submit" */ }
+					<form action="" onSubmit={ handleSubmit }>
```

Comment:
Nice! 👍  But does this require `action=""` to exist for the keyboard to hide? If not `action` attribute can be left out completely for HTML5. https://stackoverflow.com/questions/1131781/is-it-a-good-practice-to-use-an-empty-url-for-a-html-forms-action-attribute-a/1132015#1132015

Reply:
The `action` attribute it's not for the keyboard to hide but for the "search" label to appear instead of "return" on the on-screen keyboard starting with iOS 8.

Reply:
Good to know. TIL! :)

Reply:
Might be good note to add in comment above the line? "this" term in the comment currently makes it seem like everything on that line is about hiding keyboard.